packages/react/test/specs/commonTests/handlesAccessibility.tsx

```diff
@@ -8,8 +8,8 @@ import { FocusZone, FOCUSZONE_WRAP_ATTRIBUTE } from '@fluentui/react-bindings'
 import * as React from 'react'
 import * as keyboardKey from 'keyboard-key'
 
-import { mountWithProviderAndGetComponent, mountWithProvider } from 'test/utils'
-import { UIComponent } from 'src/utils'
+import { mountWithProviderAndGetComponent, mountWithProvider } from '../../utils'
+import { UIComponent } from '@fluentui/react/src/utils'
```

Comment:
```suggestion
import { UIComponent } from '../../src/utils'
```

Can we use instead for unexported things?

Reply:
I think it's kind of strange to have a combination of relative path imports and package name imports into the same code, and it might not be clear to a developer coming into the code which import type to use when. (Though you could make the same argument about clarity with when to use `@fluentui/react` vs paths like `@fluentui/react/src/whatever`.)